app/javascript/components/history/HistoryComponent.js

```diff
@@ -21,6 +21,14 @@ class HistoryComponent extends React.Component {
       filteredHistories: this.props.histories,
       pageOfHistories: [],
     };
+    this.historyCreators = [
+      {
+        label: 'History Creator',
+        options: _.uniq(props.histories.map(x => x.created_by)).map(x => {
+          return { value: x, label: x };
+        }),
+      },
+    ];
 
     this.filterOptions = [
```

Comment:
I haven't been on the past few PRs for this component, so this is the first time I'm really seeing this code. I think we should probably rename this variable to be `typeFilterData`, since there are two filters now. `historyCreators` variable could be consistently called `creatorFilterData` as well. Just to clear it up a bit.